BUILD

```diff
@@ -0,0 +1,131 @@
+repo = ["WORKSPACE",
+         ".bazelrc",
+         "//scala:sources",
+         "//scala/support:sources",
+         "//src/java/com/google/devtools/build/lib:sources",
+         "//src/java/io/bazel/rulesscala/io_utils:sources",
+         "//src/java/io/bazel/rulesscala/jar:sources",
+         "//src/java/io/bazel/rulesscala/scalac:sources",
+         "//src/java/io/bazel/rulesscala/worker:sources",
+         "//src/scala:sources",
+         "//src/scala/io/bazel/rules_scala/scrooge_support:sources",
+         "//src/scala/io/bazel/rules_scala/tut_support:sources",
+         "//src/scala/scripts:sources",
+         "//test:sources",
+         "//test/data:sources",
+         "//test/src/main/resources/scala/test:sources",
+         "//test/src/main/scala/scala/test/srcjars:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2/thrift3:sources",
+         "//test/src/main/scala/scala/test/twitter_scrooge/thrift/thrift2/thrift4:sources",
+         "//test/tut:sources",
+         "//test_expect_failure/disappearing_class:sources",
+         "//test_expect_failure/scala_library_suite:sources",
+         "//test_expect_failure/transitive/java_to_scala:sources",
+         "//test_expect_failure/transitive/scala_to_java:sources",
+         "//test_expect_failure/transitive/scala_to_scala:sources",
+         "//thrift:sources",
+         "//tut_rule:sources",
+         "//twitter_scrooge:sources"]
+
+additions = [] 
+
+sh_test(
+    name = "build_targets_under_tests_package",
+    size = "large",
+    data = repo,
```

Comment:
Adding the `local_jdk://jdk` was easy and we made some progress (It fails on another thing right now which is - it claims that it cannot load the main class JavaBinary... ([look in travis](https://travis-ci.org/bazelbuild/rules_scala/builds/207059839)).

As for "bazel itself as dependency with --javabase" - I tried to adjust Bazel's testenv.sh to our repository but it seems to have some dependencies on other packages that I couldn't find. I tried to simply add the `--javabase <path>` to the `.bashrc` but it did not make any different in the result...